airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionUpdaterWorkflow.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.scheduling;
+
+import io.airbyte.workers.temporal.scheduling.state.WorkflowState;
+import io.temporal.workflow.QueryMethod;
+import io.temporal.workflow.SignalMethod;
+import io.temporal.workflow.WorkflowInterface;
+import io.temporal.workflow.WorkflowMethod;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@WorkflowInterface
+public interface ConnectionUpdaterWorkflow {
```

Comment:
I still think the name for this workflow could be improved. Can punt to after merging though.

Reply:
I changed it to connection manager, I choose that since it is doing more than just sync (update/delete... ). Does it sounds ok to you.

Reply:
sounds good!